sdlib/d/gc/tcache.d

```diff
@@ -283,3 +366,57 @@ unittest makeRange {
 	checkRange(ptr[1 .. 7], 0, 0);
 	checkRange(ptr[1 .. 8], 8, 8);
 }
+
+unittest appendableAlloc {
```

Comment:
This test case misses the use case completely: appending to an existing allocation (or not, when it isn't possible).